system/include/emscripten/stack.h

```diff
@@ -26,17 +29,20 @@ uintptr_t emscripten_stack_get_base(void);
 // emscripten_stack_get_base().
 uintptr_t emscripten_stack_get_end(void);
 
+// Setup internal base/end values based on initial (compile time or intantiate
+// time values).
```

Comment:
I'm not sure what the parentheses here mean? is it that it sets up values on one or the other of the two?